achievements/666.achievement.js

```diff
@@ -0,0 +1,38 @@
+var sixes = {
+  name: 'I am the devil I like metal',
+  check: function(pullRequest, shall) {
+    if (isPrime(pullRequest.number)) {
+
+      var achieve = {
+        avatar: 'images/achievements/666.achievement.jpg',
+        name: 'optimus prime',
+        short: 'You\'re pull request was develish',
+        description: [
+          'A pull request with the number of the beast ',
+          pullRequest.number
+        ].join(''),
+        relatedPullRequest: pullRequest.id
+      };
+      shall.grant(pullRequest.creator.username, achieve);
+    }
+  }
+};
+
+function hasSix(n) {
```

Comment:
I'd call it with something more related to the achievement, like  function isDevilNumber (n) { // ... }